db/migrations/20230207085030_users.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+ CREATE TABLE users (
+    id INT NOT NULL AUTO_INCREMENT ,
+    last_name VARCHAR(20) NOT NULL,
+    first_name VARCHAR(20) NOT NULL,
+    email VARCHAR(320) UNIQUE NOT NULL,
+    password BINARY(60) NOT NULL ,
+    skin_type VARCHAR(20) NULL ,
+    point DECIMAL(8, 0) ,
```

Comment:
다음과 같이 수정하여 유저가 생성될 때마다 자동으로 포인트를 부여할 수 있게 해주세요!
```suggestion
    point DECIMAL(10, 2) NOT NULL DEFAULT 10000000,
```